### PR TITLE
Migrate from glog to klog

### PR DESCRIPTION
**Pull request by ahmetb**
Kubernetes ecosystem has moved to klog. This is more actively maintained
than glog. We might find caveats and gotchas along the way, but seems to be
working fine locally.

Earlier we get the better as it is very likely to incur merge conflicts.
/kind cleanup

### CODE REVIEW

**Comment by ahmetb** [comment]
Also apparently there's a go-logr implementation at https://github.com/kubernetes/klog/tree/master/klogr which looks promising https://github.com/kubernetes/klog/blob/master/examples/klogr/main.go.

File: cmd/generate-plugin-overview/go.mod [hunk 1]
```diff
@@ -4,5 +4,5 @@ go 1.12
 
 require (
 	github.com/pkg/errors v0.8.1 // indirect
-	sigs.k8s.io/krew v0.2.2-0.20190724210953-8867a94acd6d
+	sigs.k8s.io/krew v0.3.2
```
Comment (corneliusweig) [inline cmd/generate-plugin-overview/go.mod]: This _could_ have been in a different PR, but I think it's fine..

File: cmd/validate-krew-manifest/main.go [hunk 2]
```diff
@@ -40,29 +40,29 @@ var flManifest string
...
 	flag.StringVar(&flManifest, "manifest", "", "path to plugin manifest file")
-	if err := flag.Set("logtostderr", "true"); err != nil {
-		fmt.Printf("can't set log to stderr %+v", err)
-		os.Exit(1)
-	}
 }
 
 func main() {
 	// TODO(ahmetb) iterate over glog flags and hide them (not sure if possible without using pflag)
+	klog.InitFlags(nil)
```
Comment (corneliusweig) [inline cmd/validate-krew-manifest/main.go]: I looks a bit inconsistent that for the main command the setup is done in `init` whereas for validate-krew-manifest it's done in `main()`.

File: cmd/validate-krew-manifest/main.go [hunk 3]
```diff
@@ -40,29 +40,29 @@ var flManifest string
...
 func init() {
 	flag.StringVar(&flManifest, "manifest", "", "path to plugin manifest file")
-	if err := flag.Set("logtostderr", "true"); err != nil {
-		fmt.Printf("can't set log to stderr %+v", err)
-		os.Exit(1)
-	}
 }
 
 func main() {
 	// TODO(ahmetb) iterate over glog flags and hide them (not sure if possible without using pflag)
```
Comment (corneliusweig) [inline cmd/validate-krew-manifest/main.go]: I think it's time to resolve this TODO. Lets do that in a follow-up PR.

**Pull request merged**